### PR TITLE
Redirect related domains

### PR DESCRIPTION
**Pull request by mbrandonw**
redirects hosts that are not in an allow list to `www.pointfree.co`

### CODE REVIEW

File: Sources/PointFree/Routes.swift [hunk 1]
```diff
@@ -55,6 +56,38 @@ private func route<I, A, Route>(
     }
 }
 
+private let relatedDomains = [
+  "pointfree.co",
```
Comment (stephencelis) [inline Sources/PointFree/Routes.swift]: What if we did the reverse and redirect anything that doesn't match to www.pointfree.co?

``` swift
private let hosts = [
  "www.pointfree.co",
  "localhost",
  "0.0.0.0",
  "127.0.0.1",
]
```

Comment (mbrandonw) [reply Sources/PointFree/Routes.swift]: Good idea! Will do

**Comment by mbrandonw** [comment]
@stephencelis ok done! give it another look!

File: Tests/PointFreeTests/LaunchSignupTests.swift [hunk 2]
```diff
@@ -19,7 +19,7 @@ class LaunchSignupTests: TestCase {
   }
 
   func testHome_SuccessfulSignup() {
-    let request = URLRequest(url: URL(string: "/?success=true")!)
+    let request = URLRequest(url: URL(string: "http://localhost/?success=true")!)
```
Comment (stephencelis) [inline Tests/PointFreeTests/LaunchSignupTests.swift]: Might be good to hardcode a port in here to be thorough! Otherwise good!

**Pull request merged**